wazuh/wazuh-kibana-app
# Fix agent deployment instructions for HP-UX and Solaris

### Description

Fixed agent deployment instructions for HP-UX and Solaris.
 
This pull request is going to be merged into the main branch used for this epic [#4931](https://github.com/wazuh/wazuh-kibana-app/pull/4933)

### Issues Resolved

#4926 

### Evidence

<img width="1789" alt="image" src="https://user-images.githubusercontent.com/99441266/207415342-5d0fa9bf-13c2-4011-ba7b-edf6750634ec.png">

<img width="1789" alt="image" src="https://user-images.githubusercontent.com/99441266/207415422-c55e6517-1b74-41c9-95d8-2ad85a831d1f.png">

### Test

* Click on agents
* Click on deploy new agent
* The sections for setting the WAZUH_MANAGER, WAZUH_AGENT_GROUP and WAZUH_AGENT_NAME variables should not appear.

### Check List
